docs/testing/multiple_files_changed.md

```diff
@@ -0,0 +1,135 @@
+# Multiple files changed
+
+Sometimes Rector does changes in multiple files. How to test it?
+
+## Rector
+Add file with content in Rector:
+```
+$addedFileWithContent = new AddedFileWithContent($filePath, $content);
```

Comment:
```suggestion
```php
$addedFileWithContent = new AddedFileWithContent($filePath, $content);
```

It might be helpful to use FQN classes with use imports, as often people copy-paste code chunks and assume the classes are in their full form. Imports in code snippets might avoid some WTFs :)

Reply:
done